### ISSUE TITLE
My page is not opening

### ISSUE THREAD

**Issue by rbhat0987** [facebook/augmented-traffic-control#65]
I have done all the setting as shown in the read file.
Continuously getting the below attached error 
kindly help

![atc_screen](https://cloud.githubusercontent.com/assets/11641859/6818653/e420a1ee-d2d9-11e4-905e-77048e7d8697.png)

**Comment by chantra** [comment]
Hi @rbhat0987 unfirtunately, we do not support python 3 yet. While the print issue may be quite easy to fix, there would be most likely a slew of other issues. Can you get a python 2.7 install alongside python 3.4 ?

**Comment by rbhat0987** [comment]
yeh sure let me try again
thanks for fast response

**Comment by chantra** [comment]
@rbhat0987 Also, maybe I should have been a tad more descriptive, but we only support Linux. While there is a good chance that you can run the web interface on windows, the `atcd` daemon will not run on windows,

**Comment by chantra** [comment]
Marking as `invalid` for now as there is no plan to support windows.

**Comment by rbhat0987** [comment]
You dont need to worry about windows yes it works properly on the windows with the config you have provided...about atcd i am still figuring it out...thanks

**Comment by chantra** [comment]
@rbhat0987 what are you trying to figure out regarding `atcd`? this will simply not run on windows because it depends on linux specific utilities like iptables and the netlink subsystem.

**Comment by rbhat0987** [comment]
I dont have access to my routers have asked my admin for the same
